packages/phrases/src/locales/pt-pt/translation/admin-console/sign-in-exp.ts

```diff
@@ -0,0 +1,116 @@
+    dark_logo_image_url_placeholder: 'https://your.cdn.domain/logo-dark.png',
+    slogan: 'Slogan',
+    slogan_placeholder: 'Liberte a sua criatividade',
+  },
+  sign_in_methods: {
+    title: 'MÉTODOS DE LOGIN',
+    primary: 'Método de login principal',
+    enable_secondary: 'Ativar login secundário',
+    enable_secondary_description:
+      "Depois de ativado, a sua app oferecerá suporte a mais métodos de login além do principal. ",
+    methods: 'Método de login',
+    methods_sms: 'SMS',
+    methods_email: 'Email',
+    methods_social: 'Rede social',
+    methods_username: 'Utilizador e password',
+    methods_primary_tag: '(Primário)',
+    define_social_methods: 'Definir métodos de login social',
+    transfer: {
+      title: 'Conectores sociais',
+      footer: {
+        not_in_list: 'Não está na lista?',
+        set_up_more: 'Configurar mais',
+        go_to: 'conectores sociais ou vá para a seção "Conectores".',
+      },
+    },
+  },
+  others: {
+    terms_of_use: {
+      title: 'TERMOS DE USO',
+      enable: 'Ativar termos de uso',
+      description: 'Adicione os termos legais para uso do seu produto',
+      terms_of_use: 'Termos de uso',
+      terms_of_use_placeholder: 'https://your.terms.of.use/',
+      terms_of_use_tip: 'URL dos termos de uso',
+    },
+    languages: {
+      title: 'LÍNGUAS',
+      mode: 'Modo de idioma',
+      auto: 'automático',
+      fixed: 'Fixo',
+      fallback_language: 'Idioma fallback',
+      fallback_language_tip:
+        'Qual idioma usar se o Logto não encontrar o idioma requisitado.',
+      fixed_language: 'Idioma fixo',
+    },
+    authentication: {
+      title: 'AUTENTICAÇÃO',
+      enable_create_account: 'Permitir criar conta?',
+      enable_create_account_description:
+        'Ativa ou desativa a criação de contas. Depois de desativado, os seus clientes não poderão criar contas por meio da pagina de login, mas você poderá adicionar utiizadores na Consola de Administrador.',
+    },
+  },
+  setup_warning: {
+    no_connector: '',
+    no_connector_sms:
+      'Ainda não configurou um conector de SMS. A experiência de login não será ativada até que conclua as configurações primeiro. ',
+    no_connector_email:
+      'Ainda não configurou um conector de Email. A experiência de login não será ativada até que conclua as configurações primeiro. ',
+    no_connector_social:
+      'Ainda não configurou um conector social. A experiência de login não será ativada até que conclua as configurações primeiro. ',
+    no_added_social_connector:
+      'Configurou alguns conectores sociais agora. Certifique-se de adicionar alguns a experiência de login.',
+  },
+  save_alert: {
+    description:
+      'Está alterando os métodos de login. Isso afetará alguns dos seus utilizadoress. Tem a certeza que deseja fazer isso?',
+    before: 'Antes',
+    after: 'Depois',
+  },
+  preview: {
+    title: 'Pre-visualização do login',
+    dark: 'Tema escuro',
+    light: 'Tema claro',
```

Comment:
How about just `Escuro` and `Claro`?

Reply:
Oh sorry. I asked this because I saw the dropdown in the UI and it looked no good because the value is a bit longer than expected.
But I don't want it to be interpreted bad. You can keep it and we can consider improve the UI by increasing the width of the preview panel

Reply:
OK, go go go 🚀